modules/ROOT/pages/oracle/oracle-ebs-connector.adoc

```diff
@@ -254,7 +254,7 @@ image::oracle/oracle-ebs-select-connector.png[Select connector as a Design Cente
 
 === Add Libraries for This Connector
 
-To subsribe to business events or to the XML Gateway queue, the JMS Configuration requires two external libraries:
+To subscribe to business events or to the XML Gateway queue, the JMS Configuration requires two external libraries:
```

Comment:
Why is "configuration" capitalized in "JMS Configuration"? I looked this up but could not find anything definitive.